Gemfile.lock

```diff
@@ -0,0 +1,60 @@
+PATH
```

Comment:
I don't think lockfiles should be included in gems: could we ignore this? They should be testable with whatever is the latest.